go.mod

```diff
@@ -1,92 +1,70 @@
 module github.com/kelindar/talaria
 
-go 1.17
+go 1.18
 
 // should be removed once crphang/orc is up to date with upstream orc
 replace github.com/crphang/orc => github.com/tardunge/orc v0.0.7
 
-replace github.com/dgraph-io/badger/v3 v3.2103.1 => github.com/talariadb/badger/v3 v3.2103.1-with-arm-fix
```

Comment:
Are we not using this anymore?

Reply:
not closing the mr. but made relevant changes.